src/fides/api/schemas/connection_configuration/connection_secrets_rds_mysql.py

```diff
@@ -0,0 +1,31 @@
+from pydantic import Field
+
+from fides.api.schemas.base_class import NoValidationSchema
+from fides.api.schemas.connection_configuration.connection_secrets_base_aws import (
+    BaseAWSSchema,
+)
+
+
+class RDSMySQLSchema(BaseAWSSchema):
+    """
+    Schema to validate the secrets needed to connect to a RDS MySQL Database
+    """
+
+    username: str = Field(
+        default="fides_explorer",
+        title="Username",
+        description="The user account used to authenticate and access the databases.",
+    )
+    region: str = Field(
+        title="Region",
+        description="The AWS region where the RDS instances are located.",
+    )
+    ca_cert_url: str = Field(
```

Comment:
knowing what i know about how this field is used downstream in the monitor, and the fact that it's user-input data - i think we need some more validation on this field. at a minimum, we should be using the `AnyHttpUrl` pydantic class (or even better, using @pattisdr 's `AnyHttpUrlString` in `fideslang) to put some bounds on this field.

additionally, we should probably perform some more checks on the URL before we fetch it and save it to the filesystem downstream. i'm not exactly sure the best way to check that, but it feels like it'd be wise...

if you feel like it makes more sense to just hardcode the default value here rather than accept user config for it, that may be OK. i know i'd vouched for a configurable field earlier, since i thought the flexibility would be a good fallback in case a customer isn't using this global AWS cert that we expect, but i hadn't really considered the security implications. sorry to go back and forth on this! i still think the flexibility would be nice, but we need to be able to do it in a reasonably secure way, so let's weigh that accordingly.

if you do decide to remove the flexibility, i think it would be good for us to confirm with our prospective customers of this feature whether the global AWS cert we're defaulting to is one that would work for their use case 👍